slack_client.rb

```diff
@@ -0,0 +1,23 @@
+require 'json'
+require 'curb'
+
+class SlackClient
```

Comment:
Nazval by som to `SlackNotifier` a trohu specifikoval nazvy metod, aby to bol nejaky "interface" pre `Listener`. `send_message` je prilis genericka, pre DiscourseClient sa to rozbije (lebo tam treba vynutit dalsie stahovanie, db, atd, atd). V principe su tam tieto pripady, ktore by mohli byt ako metody `new_issue_not_published`, `matching_announcements_found(announcements)`. 

Nasledne bude `SlackNotifier` specializacia tohto interfacu, tak isto `DiscourseClient`, ktory bude komplikovanejsi.